docs/source/index.rst

```diff
@@ -2,33 +2,83 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Dialog Flow Framework's documentation!
-=================================================
+Dialog Flow Framework Documentation
+=====================================
 
-.. toctree::
-   :caption: Contents:
+*Date*: ??.12.2022 *Version*: 1.0.0
```

Comment:
If we want to keep this line, we will add the file into `VERSIONING_FILES = setup.py makefile docs/source/conf.py dff/__init__.py` - into makefile